lib/usd/translators/meshWriter.cpp

```diff
@@ -15,6 +15,10 @@
 //
 #include "meshWriter.h"
 
+#include "maya/MApiNamespace.h"
+#include "maya/MStatus.h"
+#include "pxr/base/tf/diagnostic.h"
```

Comment:
Angle braces for these includes, and it looks like that maybe confused clang-format, as I'd expect it to have grouped these with their respective `maya` and `pxr` groups below.

Reply:
Resolved in 2e3a363.